libraries/ms-helper/inc/heartbeat_rx.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+// RX handler for heartbeats.
+//
+// Requires CAN to be initialized.
+//
+// The concept is to simplify registering a CANRxHandler such that it automatically replies in the
+// affirmative to the message unless a callback specifies otherwise. If no handler is specified
+// |NULL| it will always return success.
```

Comment:
Should probably mention the control flow - master sends some heartbeat request critical message that this module handles the ACKs for.

Reply:
Done.